test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -1137,6 +1137,49 @@ hw.module @XMR_src(%a : i23) -> (aa: i3) {
   hw.output %r : i3
 }
 
+// CHECK-LABEL: module SubstProbeInVerbatim1
+hw.module @SubstProbeInVerbatim1(%a: i1) -> (b: i1) {
+  // CHECK:      wire       [[PROBE_ADD:.+]];
+  // CHECK-NEXT: wire       [[PROBE_READ:.+]];
+  // CHECK-NEXT: wire       [[PROBE_REG:.+]];
+  // CHECK-NEXT: wire [0:0] [[WIRE:.+]];
+  // CHECK-NEXT: reg  [0:0] [[REG:.+]];
+
+  // CHECK:      localparam _T_2 = 1'h0;
+  // CHECK-NEXT: assign [[PROBE_READ]] = wire_2[_T_2];
+  // CHECK-NEXT: assign [[PROBE_ADD]] = a + wire_2[_T_2];
+  // CHECK-NEXT: initial begin
+  // CHECK-NEXT:   force [[PROBE_REG]] = [[REG]][_T_2];
+  %wire = sv.wire  : !hw.inout<array<1xi1>>
+  %false = hw.constant false
+  %0 = sv.array_index_inout %wire[%false] : !hw.inout<array<1xi1>>, i1
+  %1 = sv.read_inout %0 : !hw.inout<i1>
+  hw.probe @read_wire, %1 : i1
```

Comment:
Drive by comment by lurker: could you add a test with `hw.probe` with more than one value?

Reply:
I'll add, thanks! Actually I am ignoring naming by probe op which probes multiple values. 
So it would be better to emit an error for this.